modules/tinymce/src/core/main/ts/DragDropOverrides.ts

```diff
@@ -143,15 +144,18 @@ const moveGhost = (
 
   const clientHeight = contentAreaContainer.clientHeight;
   const clientWidth = contentAreaContainer.clientWidth;
+  const outerMouseY = mouseY + contentAreaContainer.getBoundingClientRect().top;
+  const outerMouseX = mouseX + contentAreaContainer.getBoundingClientRect().left;
 
   state.on((state) => {
```

Comment:
All this logic is over 250 lines now... that's a pretty red flag that this is doing too much. Is there any way we can break this up into smaller functions?